src/material-experimental/mdc-slide-toggle/slide-toggle.html

```diff
@@ -3,28 +3,28 @@
   <div [ngClass]="_classes" #switch>
     <div class="mdc-switch__track"></div>
     <div class="mdc-switch__thumb-underlay">
-      <div class="mat-mdc-slide-toggle-ripple" mat-ripple
+      <input #input class="mdc-switch__native-control" type="checkbox"
```

Comment:
Is it possible to use the [recommended DOM structure](https://material.io/develop/web/components/input-controls/switches/)? i.e. putting the input inside `.mdc-switch__thumb`

Reply:
Yeah definitely. I don't understand one thing though. It looks like the `.mdc-switch__thumb` is relatively positioned in our demo, but isn't in the official MDC demo (https://material-components.github.io/material-components-web-catalog/#/component/switch). We don't want it to be relatively positioned, but I can't find anywhere in our styles where we set that. I'm confused where this style is coming from...

Reply:
@crisbeto did you intentionally deviate from the recommended DOM here?

Reply:
I don't remember doing it on purpose. Maybe the recommended structure changed at some point?

Reply:
It looks like the `position: relative` was removed recently in MDC code. @zelliott if you rebase it should go away. My guess is they updated the DOM guidance at the same time that change was made, so we should change it to follow the new guidance